src/include/pka_cpu.h

```diff
@@ -80,14 +86,191 @@ static inline uint64_t pka_early_cpu_speed(void)
 }
 #else
 
+#ifdef __BIG_ENDIAN__
+#define WORD(x) (u16)((x)[0] + ((x)[1] << 8))
+#define DWORD(x) (u32)((x)[0] + ((x)[1] << 8) + ((x)[2] << 16) + ((x)[3] << 24))
+#define QWORD(x) (U64(DWORD(x), DWORD(x + 4)))
+#else
+#define WORD(x) (uint16_t)(*(const uint16_t *)(x))
+#define DWORD(x) (uint32_t)(*(const uint32_t *)(x))
+#define QWORD(x) (*(const uint64_t *)(x))
+#endif
 #define MAX_CLOCK_CYCLES     UINT64_MAX
 
+struct dmi_header {

```

Comment:
Please use the pka lib naming convention.

  typedef struct {
    (...)
  } pka_cpu_dmi_hdr;